AzureCalling/Controllers/StartCallViewController.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import FluentUI
+
+class StartCallViewController: UIViewController, UITextFieldDelegate {
+
+    // MARK: Properties
+    private var displayNameStackView: UIStackView!
+    private var displayNameLabel: FluentUI.Label!
+    private var captionLabel: FluentUI.Label!
+    private var displayNameTextField: IconTextField!
+    private var nextButton: FluentUI.Button!
+
+    // MARK: ViewController Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        displayNameTextField.becomeFirstResponder()
+    }
+
+    // MARK: UI layout
+    private func setupUI() {
+        view.backgroundColor = ThemeColor.lightSurfacesPrimary
+        self.navigationItem.title = "Start call"
+        setupLabels()
+        setupDisplayNameTextField()
+        setupNextButton()
+        layoutTextFieldContainer()
+        layoutMainContainer()
+    }
+
+    private func setupLabels() {
+        displayNameLabel = FluentUI.Label(style: .footnote)
```

Comment:
I have a helper extension on my working branch to make this a bit more concise.
I think moving forward, we should have some generic factory to make construction of FluentUI controls more SwiftUI-like